### ISSUE TITLE
getting blank frames in results

### ISSUE THREAD

**Issue by Suvi-dha** [xuboming8/DSTNet#10]
Hi, I am inferencing on my own dataset and followed the directory structure of the dataset. My frames are of size 1920 by 1080 which I resized to 2048 by 1024 as suggested (in the multiples of 256). I also don't have the ground truth so in the yml file I kept both the low quality and ground truth data path as same. The code then successfully ran but I am getting blank black frames as output. Here's my data config and results. 

```
datasets:
  test_1:  # the 1st test dataset
    name: GOPRO
    type: VideoRecurrentSplitClipsTestDataset
    dataroot_gt: /home/suvidha/Diffusion/ResShift/testdata/blinkit_frames/RetinexFormer_NTIRE/NTIRE/blur
    dataroot_lq: /home/suvidha/Diffusion/ResShift/testdata/blinkit_frames/RetinexFormer_NTIRE/NTIRE/blur
    cache_data: True
    io_backend:
      type: disk

    num_frame: 10
    num_overlap: 0
```

![000000](https://github.com/xuboming8/DSTNet/assets/37800843/2519f97b-09c7-4704-b1a4-fb719707c868)

**Comment by xuboming8** [comment]
If you change the dataset to GoPro while maintaining the parameter configuration, will there also be black frames?
When I train ours-L, models with higher PSNR are at risk of black frames on gopro, which may be caused by the instability of dynamic convolution.

**Comment by Suvi-dha** [comment]
thanks for your reply. I didn't test on GoPRO dataset since my goal is to make it work on my dataset. Is there a way where I can make changes to avoid the instability. here's my example frame:
![blinkit 1_frame60](https://github.com/xuboming8/DSTNet/assets/37800843/b77ebe38-cbee-46e3-8194-48812cb07436)

**Comment by xuboming8** [comment]
（1）Reducing the number of test frames（num_frame: 10） may avoid this issue.
（2）Retrain a model with lower PSNR.
（3）Try the other models trained on different dataset as provided in ./experiments.
（4）Using patch instead of the entire frame during inference.
By the way,DSTNet requires input of multiples of 16. And we usually do not use ‘resize’ to change the size of each frames.Adding padding is a more commonly used method.
e.g.（h_n = (32 - h % 32) % 32
w_n = (32 - w % 32) % 32
 input_img = torch.nn.functional.pad(input_img, (0, w_n, 0, h_n), mode='reflect')）

**Comment by Suvi-dha** [comment]
Ok thanks a lot and I appreciate your prompt reply and help. I shall try these suggestion.
